File: src/main/java/com/baulsupp/oksocial/services/spotify/SpotifyUtil.java
package com.baulsupp.oksocial.services.spotify;

import java.util.Arrays;
import java.util.Collection;
import okhttp3.Request;

public class SpotifyUtil {
  public static final Collection<String> SCOPES = Arrays.asList("playlist-read-private",
      "playlist-read-collaborative",
      "playlist-modify-public",
      "playlist-modify-private",
      "streaming",
      "ugc-image-upload",
      "user-follow-modify",
      "user-follow-read",
      "user-library-read",
      "user-library-modify",
      "user-read-private",
      "user-read-birthdate",
      "user-read-email",
      "user-top-read");

  private SpotifyUtil() {
  }

  public static Request apiRequest(String s, Request.Builder requestBuilder) {
    return requestBuilder.url("https://api.spotify.com" + s).build();
  }


}

File: src/main/kotlin/com/baulsupp/oksocial/services/spotify/SpotifyUtil.kt
package com.baulsupp.oksocial.services.spotify

import okhttp3.Request
import java.util.Arrays

object SpotifyUtil {
    val SCOPES: Collection<String> = Arrays.asList("playlist-read-private",
            "playlist-read-collaborative",
            "playlist-modify-public",
            "playlist-modify-private",
            "streaming",
            "ugc-image-upload",
            "user-follow-modify",
            "user-follow-read",
            "user-library-read",
            "user-library-modify",
            "user-read-private",
            "user-read-birthdate",
            "user-read-email",
            "user-top-read")

    fun apiRequest(s: String, requestBuilder: Request.Builder): Request {
        return requestBuilder.url("https://api.spotify.com" + s).build()
    }


}
